packages/firebase_ml_vision/lib/src/document_text_recognizer.dart

```diff
@@ -69,6 +76,27 @@ class DocumentTextRecognizer {
   }
 }
 
+/// Options for cloud document text recognizer.
```

Comment:
nit: I believe this should be a little bit more descriptive: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-useless-documentation